PluginCore/ScintillaNet/ScintillaControl.cs

```diff
@@ -725,14 +725,17 @@ private void SetLanguage(String value)
                     }
                     catch (Exception ex)
                     {
+                        String info;
                         if (lexerType == null)
                         {
-                            ErrorManager.ShowWarning($"Lexer '{lang.lexer.name}' ({lang.lexer.key}) unknown.", ex);
+                            info = String.Format("Lexer '{0}' ({1}) unknown.", lang.lexer.name, lang.lexer.key);
+                            ErrorManager.ShowWarning(info, ex);
```

Comment:
I changed it while browsing the file, and added in the first commit. That's a revert change you've commented on.
>I would prefer if you revert the changes to ScintillaControl for the time being as they are not needed here. - @Neverbirth